vars/pipelineVars.groovy

```diff
@@ -31,6 +30,10 @@ class pipelineVars implements Serializable {
     String prodCluster = "api.insights.openshift.com"
     String devCluster = "api.insights-dev.openshift.com"
 
+    String jenkinsSlaveImage = 'openshift/jenkins-slave-base-centos7:v3.11'
+    String centralCIjenkinsSlaveImage = 'docker-registry.engineering.redhat.com/centralci/jnlp-slave-base:1.5'
+    String iqeCoreImage = 'quay.io/cloudservices/iqe-core'
+
     String jenkinsSlaveIqeImage = 'jenkins-slave-iqe:latest'
```

Comment:
`jenkinsSlaveIqeImage` can be deleted from pipelineVars at this point right?

Reply:
Correct